src/input/composite_device/mod.rs

```diff
@@ -604,8 +604,11 @@ impl CompositeDevice {
         //log::trace!("Event capability: {:?}", cap);
 
         // Only send valid events to the target device(s)
-        if cap == Capability::NotImplemented {
-            log::trace!("Refusing to send 'NotImplemented' event to target devices");
+        if cap == Capability::NotImplemented || cap == Capability::Sync {
```

Comment:
Instead of blocking the events at the composite device, why don't we instead block them in the `evdev` source device? That will prevent those events from filling up the channel buffer.

Reply:
Updated